coalib/parsing/FilterHelper.py

```diff
@@ -21,34 +21,35 @@ def is_valid_filter(cls, filter):
         return filter in cls.available_filters
 
     @classmethod
-    def apply_filter(cls, filter, args, all_bears=None):
-        if all_bears is None:
+    def apply_filter(cls, filter_name, filter_args, bears=None):
+        if bears is None:
             from coalib.settings.ConfigurationGathering import (
                 get_all_bears)
-            all_bears = get_all_bears(LogPrinter())
-        if not cls.is_valid_filter(filter):
+            bears = get_all_bears(LogPrinter())
+        if not cls.is_valid_filter(filter_name):
             raise InvalidFilterException('{!r} is an invalid filter. '
                                          'Available filters: {}'.format(
-                                             filter, cls.get_all_filters_str()))
-        if not args or len(args) == 0:
-            return all_bears
-        return cls.available_filters[filter](all_bears, args)
+                                             filter_name,
+                                             cls.get_all_filters_str()))
+        if not filter_args or len(filter_args) == 0:
+            return bears
+        return cls.available_filters[filter_name](bears, filter_args)
 
     @classmethod
-    def apply_filters(cls, args, bears=None):
+    def apply_filters(cls, filters, bears=None):
         """
         Returns bears after filtering based on ``args``. It returns
         intersection of bears if more than one element is present in ``args``
         list.
 
-        :param bears: The bears to filter.
-        :param args:  List of args based on ``bears`` has to be filtered.
+        :param filters:  List of args based on ``bears`` has to be filtered.
```

Comment:
You have to reindent all parameters :/

There's actually another format which I try to make coala switch to, but our bears aren't ready yet to enforce that style:
```
description like before

even more description

:param xxx:
    4 space indent but on next line.
:return:
    stuff
```
If you like you could switch to it, but not necessary if you don't want it. However, often less lines have to be modified as indentation of params does not depend on the longest param name^^